en/docs/learn/enterprise-integration-patterns/message-transformation/envelope-wrapper.md

```diff
@@ -13,131 +13,160 @@ The Envelope Wrapper EIP allows existing systems to participate in a messaging e
 
 ## Sample scenario
 
-This example scenario receives a message with application data wrapped inside an envelope, unwraps the message, and sends it to a specific endpoint. The sender sends the request inside a SOAP envelope. Once the ESB receives the envelope, it unwraps it and sends it as a Plain Old XML (POX) request to the sample back-end Axis2 server.
+This example scenario receives a message with application data wrapped inside an envelope, unwraps the message, and sends it to a specific endpoint. The sender sends the request inside a SOAP envelope. Once the WSO2 MI receives the envelope, it unwraps it and sends it as a Plain Old XML (POX) request to the sample back-end Axis2 server.
 
-The diagram below depicts how to simulate the example scenario using the WSO2 ESB.
+The diagram below depicts how to simulate the example scenario using the WSO2 MI.
 
 <img src="{{base_path}}/assets/img/learn/enterprise-integration-patterns/message-transformation/envelope-wrapper-sample-scenario.png" style="width: 70%;" alt="Envelope wrapper sample scenario">
 
 Before digging into implementation details, let's take a look at the relationship between the example scenario and the Envelope Wrapper EIP by comparing their core components.
 
-| Envelope Wrapper EIP (Figure 1) | Envelope Wrapper Sample Scenario (Figure 2)  |
+| Envelope Wrapper EIP            | Envelope Wrapper Sample Scenario             |
 |---------------------------------|----------------------------------------------|
 | Wrapper                         | Stock Quote Request wrapped in SOAP          |
-| Messaging                       | System	WSO2 ESB                              |
+| Messaging                       | WSO2 MI                                      |
 | Unwrapper                       | Address Endpoint format                      |
 | Recipient                       | Stock Quote Service Instance                 |
 
 !!! note
     An alternative implementation of this EIP is to have the Address Endpoint wrap from one envelope format to another (for example, wrapping a SOAP 1.1 envelope in a SOAP 1.2 envelope).
 
-### Environment setup
+## Synapse configuration of the artifacts
+
+Given below is the synapse configuration of this sample.
+
+=== "Proxy Service"
+    ```xml
+    <?xml version="1.0" encoding="UTF-8"?>
+    <proxy name="EnvelopeUnwrapProxy" startOnLoad="true" transports="http https" xmlns="http://ws.apache.org/ns/synapse">
+        <target>
+            <inSequence>
+            <call>
+                <endpoint key="SimpleStockEp"/>
+            </call>
+            <respond/>
+            </inSequence>
+            <faultSequence>
+            <log category="INFO" level="full">
```

Comment:
Do we need to repeat the default fault sequence here? Ideally MI should use the default fault sequence. Shall we check it and update this

Reply:
We have added the synapse config that is there in the vs code. By default fault sequence is added.

Reply:
Does it add the fault sequence implementation as well? But there is no fault sequence implementation added in the proxy in the [Content Enricher](https://github.com/wso2/docs-mi/pull/745/files#diff-35155d2e290f567322b33eac376cc4edb45d093b2498efe506f567b3f4724c94R61)

Reply:
Since the added fault sequence is similar to the default fault sequence, the added fault sequence is removed.